cdap-cli/src/main/java/co/cask/cdap/cli/command/system/HelpCommand.java

```diff
@@ -225,16 +225,22 @@ private void wrappedPrint(String str, PrintStream output, int colWidth, int pref
     }
 
     int beginIdx = 0;
+    int returnIdx = 0;
```

Comment:
can you make the variable name more descriptive? I don't see it being *return*ed.